app/Handlers/Utility/CurrencyHandler.php

```diff
@@ -0,0 +1,133 @@
+
+    /**
+     * @param JsonLoader $loader
+     */
+    public function __construct(JsonLoader $loader)
+    {
+        $this->loader = $loader;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canHandle(Event $event)
+    {
+        return
+            $event->isMessage() &&
+            ($event->isDirectMessage() || $event->mentions($this->eve->userId())) &&
+            $event->matches('/convert .+/i')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(Event $event)
+    {
+        $this->loadData();
+
+        $arguments = $this->arguments($event);
+        $content   = '`@eve convert 10 GPB to USD`';
+        $baseRate  = 1;
+
+        if ($arguments->count() == 4 && $this->validArguments($arguments)) {
+            $rates  = $this->data['rates'];
+            $result = '';
+
+            if (! array_key_exists($arguments[1], $rates) && ! array_key_exists($arguments[3], $rates)) {
+                return;
+            }
+
+            if (array_key_exists($arguments[1], $rates) && array_key_exists($arguments[3], $rates)) {
+                $result = number_format($arguments[0] * $rates[strtoupper($arguments[3])], 2);
+            }
+
+            if (array_key_exists($arguments[1], $rates) && ! array_key_exists($arguments[3], $rates)) {
+                if ($rates[strtoupper($arguments[1])] > $baseRate) {
+                    $result = number_format($arguments[0] * $rates[strtoupper($arguments[1])], 2);
+                } else {
+                    $result = number_format($arguments[0] / $rates[strtoupper($arguments[1])], 2);
+                }
+            }
+
+            if (! array_key_exists($arguments[1], $rates) && array_key_exists($arguments[3], $rates)) {
+                if ($rates[strtoupper($arguments[3])] > $baseRate) {
+                    $result = number_format($arguments[0] / $rates[strtoupper($arguments[3])], 2);
+                } else {
+                    $result = number_format($arguments[0] * $rates[strtoupper($arguments[3])], 2);
+                }
+            }
+
+            $content = sprintf(
+                "<@%s> %s%s is around %s%s",
+                $event->sender(),
+                $this->symbol($arguments[1]),
+                $arguments[0],
+                $this->symbol($arguments[3]),
+                $result
+            );
+        }
+
+        $this->send(
+            Message::saying($content)
+            ->inChannel($event->channel())
+            ->to($event->sender())
+        );
+    }
+
+    /**
+     * @param  $arguments
+     *
+     * @return boolean
+     */
+    private function validArguments($arguments) {
+        return
+            preg_match('/[A-Z]{3}/', $arguments[1]) &&
+            trim($arguments[2] == 'to') &&
+            preg_match('/[A-Z]{3}/', $arguments[3])
+        ;
+    }
+
+    /**
+     * @param  string $currency
+     *
+     * @return string
+     */
+    private function symbol(string $currency)
```

Comment:
Consider renaming this method to make it more obvious what it's doing... Something like `symbolForCurrency`